src/main/kotlin/uk/gov/justice/digital/hmpps/hmppsinterventionsservice/service/ReferralService.kt

```diff
@@ -45,9 +46,9 @@ class ReferralService(
     return referralRepository.findBySentAtIsNotNull().map { SentReferralDTO.from(it) }
   }
 
-  fun getSentReferralsForServiceProviderID(serviceProviderID: AuthGroupID): List<SentReferralDTO> {
```

Comment:
not a blocker because i see this is consistent with out methods in this class, but aren't we leaving the controllers to do all the DTO mappings?

Reply:
Conscious JIRA work was unplanned so didn't want it's scope to increase. Plus we're planning to have a specific mop up task to purge the DTOs from the business tier.